worker-pools.yml

```diff
@@ -2986,6 +3020,37 @@ pools:
                 sourceImage: <image>
               type: PERSISTENT
           machine_type: c2-standard-4
+  - pool_id: '{pool-group}/t-linux-2404-d2g-noscratch'
```

Comment:
I implemented a "suffix" mechanism in some other pools to handle cases like this "noscratch" variant. It might be worth re-using that to cut down on config duplication.. however in this case I think it likely would need some code changes to go along with it. So feel free to leave it as a follow-up.

Reply:
Thanks, will keep that in mind.